### PR TITLE
Draft: add Privacy Statement

### PR DESCRIPTION
**Pull request by KevinZonda**

### CODE REVIEW

**Comment by DuckSoft** [comment]
We are missing something here: Will we share our data with some third-parties?

**Comment by KevinZonda** [comment]
> We are missing something here: Will we share our data with some third-parties?

Will do. We share nothing with 3rd. (x

**Comment by KevinZonda** [comment]
@DuckSoft How about ca96e0b ?

File: en-us/privacy.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+- Improve and develop our products.
+- Personalize our products and make recommendations.
+
+## How to access and control your personal data
+
+You can also make choices about the collection and use of your data by FastGit. You can control your personal data that FastGit has obtained, and exercise your data protection rights, by contacting FastGit. In some cases, your ability to access or control your personal data will be limited, as required or permitted by applicable law. How you can access or control your personal data will also depend on which products you use.
+
+## Cookies and similar technologies
+
+We promise not to use cookies. If the Cookie is configured for Github, we try not to interfere.
```
Comment (b1f6c1c4) [inline en-us/privacy.md]:
```suggestion
We promise NOT to use any cookies. However, GitHub may use cookies for their own purposes, in which case we are unable and prefer not to interfere. Please refer to [Cookies on GitHub](https://docs.github.com/en/github/site-policy/github-subprocessors-and-cookies#cookies-on-github) for more information.
```

File: en-us/privacy.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Privacy Statement
+
+Your privacy is important to us. This privacy statement explains the personal data FastGit processes, how FastGit processes it, and for what purposes.
+
+## Personal data we collect
+
+FastGit collects data from you, we get some of it by collecting data about your interactions usage, and experiences with our products. The data we collect depends on the context of your interactions with FastGit and the choices you make, including your privacy settings and the products and features you use.
+
+For the running of the FastGit core service, we promise to collect only the standard NGINX access logs (contain your IP address, access path and request size, etc.) . For more information on access logs, see [NGINX log module](https://nginx.org/en/docs/http/ngx_http_log_module.html) and our open source [NGINX profile](https://github.com/fastgitorg/NGINX-conf) .
```
Comment (b1f6c1c4) [inline en-us/privacy.md]:
```suggestion
For the running of the FastGit core service, we promise to collect only the standard NGINX access logs (contain your IP address, access path, request size, etc.) For more information on access logs, see [NGINX log module](https://nginx.org/en/docs/http/ngx_http_log_module.html) and our open source [NGINX profile](https://github.com/fastgitorg/NGINX-conf).
```

File: en-us/privacy.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+## How we use personal data
+
+FastGit uses the data we collect to provide you with rich, interactive experiences. In particular, we use data to:
+
+- Provide our products, which including data analysis, server sourcing as well as providing support.
+- Improve and develop our products.
+- Personalize our products and make recommendations.
```
Comment (b1f6c1c4) [inline en-us/privacy.md]:
```suggestion
FastGit uses the data we collect to provide you with rich, interactive experiences. In particular, we use your data to:

- Provide our products, which including data analysis, server sourcing as well as providing support;
- Improve our products and develop new ones;
- Personalize our products and make recommendations for you.
```

File: en-us/privacy.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+## How to access and control your personal data
```
Comment (b1f6c1c4) [inline en-us/privacy.md:23]:
```suggestion
## How do you access and control your personal data
```

File: en-us/privacy.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
+# Privacy Statement
+
+Your privacy is important to us. This privacy statement explains the personal data FastGit processes, how FastGit processes it, and for what purposes.
+
+## Personal data we collect
+
+FastGit collects data from you, we get some of it by collecting data about your interactions usage, and experiences with our products. The data we collect depends on the context of your interactions with FastGit and the choices you make, including your privacy settings and the products and features you use.
+
+For the operation of our core FastGit services, we promise to collect only the standard NGINX access logs (contain your IP address, access path and request size, etc.) . For more information on access logs, see [NGINX log module](https://nginx.org/en/docs/http/ngx_http_log_module.html) and our open-source [NGINX profile](https://github.com/fastgitorg/NGINX-conf) .
```
Comment (b1f6c1c4) [inline en-us/privacy.md]:
```suggestion
For the operation of our core FastGit services, we promise to collect only the standard NGINX access logs (contain your IP address, access path, request size, etc.) For more information on access logs, see [NGINX log module](https://nginx.org/en/docs/http/ngx_http_log_module.html) and our open-source [NGINX profile](https://github.com/fastgitorg/NGINX-conf).
```

**Review by b1f6c1c4 — CHANGES_REQUESTED** [review]
Documents in English have been revised. Documents in other languages shall adjust accordingly.
Pay **special** attention to whether my modifications alter your original meaning.

File: en-us/privacy.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+FastGit uses the data we collect to provide you with rich, interactive experiences. In particular, we use data to:
+
+- Provide our products, which including data analysis, server sourcing as well as providing support.
+- Improve and develop our products.
+- Personalize our products and make recommendations.
+
+## Share personal data
+
+We don't share your personal data with the third-party.
```
Comment (b1f6c1c4) [inline en-us/privacy.md]:
```suggestion
## Sharing your personal data

We do NOT share your personal data with any third parties.
```

File: en-us/privacy.md (hunk 4 above)
Comment (KevinZonda) [reply en-us/privacy.md:23]: Microsoft uses the same sentence. <https://privacy.microsoft.com/en-gb/privacystatement>

Comment (b1f6c1c4) [reply en-us/privacy.md:23]: Someone else using it does justify its correctness. Anyway, I always prefer formal expressions in legal context.

File: en-us/privacy.md (hunk 3 above)
Comment (KevinZonda) [reply en-us/privacy.md]: Is `new ones` better than `our products`?

**Pull request merged**